CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Compatible changes
 
+
+## 7.1.0 - 2021-11-29
+
+### Compatible changes
+
+- Disable `RSpec/ExpectChange`
+
+
+
```

Comment:
nit: You can check the number of newlines for other entries in the CHANGELOG, I think we only used two here.

Reply:
I only checked the last two entries and they both had 3 newlines. But you're right, older ones had only 2 newlines. I'll use 2 here and also remove one line for the last two entries in order to keep the style consistent